app/controllers/exports_controller.rb

```diff
@@ -0,0 +1,11 @@
+class ExportsController < ApplicationController
```

Comment:
I think you should `include ActionController::Live` like [in the backups controller](https://github.com/on-site/StockAid/blob/master/app/controllers/backups_controller.rb#L2). It adds additional functionality specifically for streaming a response like you are doing. Digging into the Rails code a bit, it seems like this would still work as is, but I noticed when `ActionController::Live` is included, at the very least, a no-cache header gets sent, and additional functionality may be added later that could be important to live streaming a response.